client/components/duplicate-notice/index.tsx

```diff
@@ -68,7 +68,7 @@ function DuplicateNotice( {
 	return (
 		<InlineNotice
 			status="warning"
-			icon={ true }
```

Comment:
Making the "duplicated payment method notice" to not have an icon, to be consistent with the other notices displayed in the settings.